macos/ONLYOFFICE/update/updates/ONLYOFFICE/changes/7.5.0/changes.html

```diff
@@ -91,46 +91,71 @@ <h2>New Features</h2>
 
         <h4>All Editors</h4>
         <ul>
-            <li>TODO</li>
+            <li>Added the support for >200% (225, 250, 275, 300, 350, 400, 450, 500 interface scaling)</li>
+            <li>In the <code>“Open Recent”</code> section, added files in the format corresponding to the editor from the “Recent files” list of the application</li>
```

Comment:
<li>Files from the "Recent files" list of the application have been added to the "File" > “Open Recent” section,  according to the editor formats</li>